src/rest.ts

```diff
@@ -109,18 +130,20 @@ function connect(uid) {
 
                     // process the initial response data
                     if (body)
-                        onDataMessage({ detail: body });
+                        onDataMessage(_ev, body);
                 } else {
-                    manage_error(reject, { reason: headers.split('\r\n')[0] }, body);
+                    // empty body Should not Happen™, would be a podman bug
+                    const body_text = body ? decoder.decode(body) : "(empty)";
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2214-e5ac0ddf-20250801-075515-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)